File: 1021-remove-outermost-parentheses/1021-remove-outermost-parentheses.py
class Solution:
    def removeOuterParentheses(self, s: str) -> str:
        open = 0 
        ans=""
        for ch in s:
            if(ch=='('):
                open+=1
                if(open>1):
                    # list have append fuct. and "".join(lst) 
                    ans+='('
            else:
                if(open>1):
                    ans+=')'
                open-=1
        return ans
            
        

File: 1021-remove-outermost-parentheses/1021-remove-outermost-parentheses.java
class Solution {
    public String removeOuterParentheses(String s) {
        int open=0;
        StringBuilder sb= new StringBuilder("");
        for(char ch:s.toCharArray()){
            if(ch=='('){
                open++;
                if(open>1) sb.append('(');
            }else{
                if(open>1){
                    sb.append(')');
                }
                open--;
            }
        }
        return sb.toString();
    }
}